webserver/grpc/src/main/java/io/helidon/webserver/grpc/GrpcProtocolHandler.java

```diff
@@ -63,11 +85,13 @@ class GrpcProtocolHandler<REQ, RES> implements Http2SubProtocolSelector.SubProto
     private final StreamFlowControl flowControl;
     private Http2StreamState currentStreamState;
     private ServerCall.Listener<REQ> listener;
-    private ServerCall<REQ, RES> serverCall;
 
-    private long length;
-    private boolean isCompressed;
-    private BufferData entityBytes = null;
+    private BufferData entityBytes;
+    private final AtomicInteger numMessages = new AtomicInteger();
```

Comment:
please move final fields with the other final fields (above non-finals)

Reply:
Fixed